### PR TITLE
Lowrisc ip update

### PR DESCRIPTION
**Pull request by GregAC**
Vendors in the latest IP from opentitan.

prim_count has changed so a small tweak to `ibex_lockstep` was required. There are no functional differences.

### CODE REVIEW

**Comment by msfschaffner** [comment]
Does it maybe make sense to wait until the CIP lib changes have been merged here https://github.com/lowRISC/opentitan/pull/21226? Just because all of that stuff gets vendored.

**Comment by GregAC** [comment]
> Does it maybe make sense to wait until the CIP lib changes have been merged here [lowRISC/opentitan#21226](https://github.com/lowRISC/opentitan/pull/21226)? Just because all of that stuff gets vendored.

I don't think it matters, Ibex doesn't use those parts of the CIP lib (the icache testbench uses some of it, but it doesn't care about interrupts).

**Review by marnovandermaas — APPROVED** [review]
Had a quick look through the vendored in IP and I think the only thing is the update to `prim_count`.

**Pull request merged**
